README.md

```diff
@@ -125,7 +127,7 @@ See [API Documentation](https://github.com/orbitdb/ipfs-log/tree/master/API.md)
 
 - [Log](https://github.com/orbitdb/ipfs-log/tree/master/API.md#log)
   - [Constructor](https://github.com/orbitdb/ipfs-log/tree/master/API.md##constructor)
-    - [new Log(ipfs, [id])](https://github.com/orbitdb/ipfs-log/tree/master/API.md##new-log-ipfs-id)
+    - [new Log(ipfs, access, identity, [id])](https://github.com/orbitdb/ipfs-log/tree/master/API.md##new-log-ipfs-id)
```

Comment:
Should `id` here be `logId` and access be in the optional field? Ie. `new Log(ipfs, identity, [logId, access])...`?